src/bottom.rs

```diff
@@ -119,4 +119,17 @@ mod tests {
             "がんばれ",
         );
     }
+
+    #[test]
+    fn test_embedded_nul_byte() {
```

Comment:
This should be `test_embedded_null_byte`

Reply:
fixed